src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Consider adding a comment to describe the purpose of the event listener for 'DOMContentLoaded'. This will help future developers understand why this listener is necessary.